### PR TITLE
[Simulator] Hide the Balance button

### LINKED ISSUE #1074: [Simulator] allow token balance chart toggle on/off from visibility
Default state - token balance chart visible [mockup](https://www.figma.com/file/YpsatcY9jtxJr0SJwlFYNj/Carbon-Screen-Design?type=design&node-id=12513%3A51250&mode=design&t=rITpzRh3QFG41udK-1)
On "eye" icon click- hide the token balance chart [mockup](https://www.figma.com/file/YpsatcY9jtxJr0SJwlFYNj/Carbon-Screen-Design?type=design&node-id=13543%3A46336&mode=design&t=rITpzRh3QFG41udK-1)
On page refresh- show token balance chart again

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**
fix #1074 

- [x] Create a button to hide balance
- [x] Hide flickering when chart is transitioning 

https://github.com/bancorprotocol/carbon-app/assets/8143464/1917e270-2a6d-42db-8c1b-abaf3f8d0a0d

- [x] Add focus & keyboard navigation for summary

https://github.com/bancorprotocol/carbon-app/assets/8143464/8feff9bf-88b0-4548-a3b3-7a8c540ad41f

### CODE REVIEW

File: src/components/simulator/result/d3Charts/D3ChartSimulatorBalance.tsx [hunk 1]
```diff
@@ -177,3 +195,57 @@ const Bar = ({
...
+}
+
+const ToggleChart: FC<ToggleChartProps> = (props) => {
+  const { isVisible, setIsVisible, dms, titleWidth, marginTop } = props;
+  const toggle = () => setIsVisible(!isVisible);
+  const center = dms.width / 2;
+  const marginLeft = 12;
+
+  const scale = isVisible ? 1 : 32 / 24;
+  const x = isVisible ? center + titleWidth / 2 + marginLeft : center / 2 + 6;
```
Comment (tiagofilipenunes) [inline src/components/simulator/result/d3Charts/D3ChartSimulatorBalance.tsx:214]:
```suggestion
  const x = isVisible ? center + titleWidth / 2 + marginLeft : center -16;
```

**Review by tiagofilipenunes — APPROVED** [review]

**Comment by tiagofilipenunes** [comment]
Good job, looks great!

**Review by pingustar — CHANGES_REQUESTED** [review]
- [ ] there are super large borders around the buttons now

image
image

**Review by pingustar — DISMISSED** [review]
- [x] there are super large borders around the buttons now

image
image

---

- [x] eye icons are to close to the edge now

now:
image

before:
image

**Review by ashachaf — APPROVED** [review]

**Pull request merged**